main.go

```diff
@@ -85,6 +85,11 @@ func main() {
 			os.Exit(1)
 		}
 		providers[config.Name] = provider
+		err = provider.Init(ctx, log)
+		if err != nil {
+			log.Error(err, "unable to create provider client")
```

Comment:
we should mention which provider so the user can fix it

Reply:
+1